ChangeLog

```diff
@@ -1,3 +1,8 @@
+2015-02-20  Tamer A. Mansour  <drtamermansour@gmail.com>
+
+   * In the do.sh script, the old argument --savehash has been updated to
+--savetable
```

Comment:
Wrapped lines should be indented to match the line above. In this instance you need three space before --savetable